dashboard/app/dsl/user_id_replace_dsl.rb

```diff
@@ -0,0 +1,2 @@
+class UserIdReplaceDSL < ExternalDSL
+end
```

Comment:
That makes sense.  Follow up question here: https://github.com/code-dot-org/code-dot-org/pull/14486#issuecomment-295493760